source/DataBase/DataBaseCommon.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+struct DataBaseCommon
+{
+    static const std::vector<std::string> userRecords;
+    static const std::vector<std::string> passwordRecords;
+
+    static const std::string dataBasePath;
+    static const std::string users;
+    static const std::string passwords;
```

Comment:
Consider using `static constexpr char variable[]` or `static constexpr std::string_view` depending on your needs.

Reply:
I used std::string_view but there was a type conflict with string. That's why I used std::format with cpp20 to convert it to a string. But I don't know if it's overkill

Reply:
Yes, you can leave static strings.